### DISCUSSION TITLE
Custom domain throws ERR_CONNECTION_CLOSED on company VPN

### DISCUSSION THREAD

**Discussion by tarrcimar** [vercel/community#4459] [Help]
### Summary

My vercel site has been working fine, but recently when I try to access it using my companys VPN, it give an ERR_CONNECTION_CLOSED error, it also looks there is no SSL certificate on the site but the issue may be with my DNS settings, I don't know. I added a domain to it: myDomain.com, followed all the steps, added a CNAME record, A record, CAA record, but to no avail. myProject.vercel.app loads without any issues.What can i do here? I added the domain with goDaddy. I originally added the records through GoDaddy, but then I switched to nameservers, hoping that would fix it, unfortunately to no avail.
I would appreciate any insight

### Steps to Reproduce

- Cannot, since it is a workplace VPN

**Comment by jack-weilage** [comment]
Does what's happening on whatcom.live look similar to your problem? This same thing just started happening to me ~1 day ago.

When I run `curl -vv https://whatcom.live` I get
```
> curl -vv https://whatcom.live
*   Trying 76.76.21.21:443...
* Connected to whatcom.live (76.76.21.21) port 443 (#0)
* schannel: disabled automatic use of client certificate
* ALPN: offers http/1.1
* schannel: failed to receive handshake, SSL/TLS connection failed
* Closing connection 0
* schannel: shutting down SSL/TLS connection with whatcom.live port 443
curl: (35) schannel: failed to receive handshake, SSL/TLS connection failed
```

**Reply by tarrcimar** [reply]
Unfortunately no, in my case it does not even list 76.76.21.21.
Though in your case it could be an SLL cert issue. It loads on both my work and personal machines (and nice site btw)

**Reply by jack-weilage** [reply]
Thanks! Digging a bit more, I think my network is at fault here? EDIT: Yep, resetting my network fixed the problem. Hope you find a solution!

**Reply by tarrcimar** [reply]
Do you have SSL certificates set up in your Vercel project? It's under DNS something

**Reply by jack-weilage** [reply]
The project is in the hobby tier, so I'm only able to use Vercel's built-in SSL afaik.
